DevEnv: Individual line option in astylerc updated

Rohan-Dah opened the pull request:
This PR is about an individual statement getting its own line as is mentioned in the style guide linked https://ardupilot.org/dev/docs/style-guide.html  

keep-one-line-statements is set to true which will make the changes as required.

rmackay9 commented:
Hi @Rohan-Dah,

If possible could you rebase on master to so that the "merge" commit disappears?  Also it would be good to change to commit title to be prefixed with "Tools: ".  Also just to be clear, I mean the commit title, not the PR title.

Rohan-Dah commented:
@rmackay9 Sure...

Rohan-Dah commented:
Hey @rmackay9 ... I've made a new PR cuz I wasn't able to edit the commit as changes I made were commited and when I tried restoring (reverting back to previous change) it lead to some crash. However, rebased it to master as per your suggestion and fixed the commit title. Will it be fine?

https://github.com/ArduPilot/ardupilot/pull/22809
